contrib/pyln-client/pyln/client/plugin.py

```diff
@@ -141,6 +141,24 @@ def set_exception(self, exc: Union[Exception, RpcException]) -> None:
     def _write_result(self, result: dict) -> None:
         self.plugin._write_locked(result)
 
+    def notify(self, message: str, level='DEBUG') -> None:
+        """Send a notification to the caller.
```

Comment:
We already have this?  It's split into the two kinds of notifications we have currently defined though: notify_message and notify_progress.

Reply:
Oh, I didn't see that it's in the `Plugin` not the `Request`, that's a really weird place for it, if we require access to the request anyway. I'll change `Plugin.notify_message` to become a wrapper of `Request.notify` and move that code there instead.

Reply:
Ah, good point.  This is a new API, so just move it.